source/contentprovider.d

```diff
@@ -154,6 +157,19 @@ class ContentProvider
 		LanguageMeta langMeta;
 		langMeta.start = ChapterAndSection(root["start"].as!string);
 
+		if (auto node = "translations" in root)
+		{
+			// YAML-D provides no convenient method to convert a node to a dictionary
```

Comment:
I looked at this (again a while ago) and I found that D-YAML uses linear search, so for now it's definitely better to keep a hashtable on our side.